src/otx/core/metrics/fmeasure.py

```diff
@@ -652,7 +654,7 @@ def __init__(
         self.cross_class_nms = cross_class_nms
         self.preds: list[list[tuple]] = []
         self.targets: list[list[tuple]] = []
-        self.num_classes: int | None = None
+        self.num_classes = num_classes
```

Comment:
@eugene123tw I think this is probably the part that relates to the issue you encountered. Please check it out.